src/pages/transactions/edit/Edit.tsx

```diff
@@ -87,8 +88,6 @@ export default function Edit() {
         toast.success('updated_transaction');
 
         $refetch(['bank_transactions']);
-
-        navigate('/transactions');
```

Comment:
Doesn't this change current app behavior? /cc @turbo124

Reply:
We should not navigate away on save.